src/components/Footer/demo.js

```diff
@@ -0,0 +1,7 @@
+// @flow
```

Comment:
Since it looks like we're going to have to add a `demo.js` with each new component, to maintain the benefits of Storybook.  Can we document what that process is and any gotchas that you know of?

Reply:
Hey @jayce-leathers I think I like your wiki suggestion, so I started adding a couple pages.

Is this intro to Storybook useful?  Let me know if I didn't answer any questions, or there's something I could improve on.

https://github.com/civic-tech-lab/commonplace-frontend/wiki/React-Storybook

Reply:
👍